src/main/common/huffman_table.c

```diff
@@ -29,261 +29,261 @@
  */
 const huffmanTable_t huffmanTable[HUFFMAN_TABLE_SIZE] = {
 //   Len    Code       Char Bitcode
```

Comment:
Realign comment or maybe just remove now that it's mostly obvious.

Reply:
Aligned